ERROR - 2025-08-02 01:53:17 --> mysqli_sql_exception: Duplicate entry 'datobar@pucesi.edu.ec' for key 'correo' in C:\xampp\htdocs\proyectoDw\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\proyectoDw\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\proyectoDw\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\proyectoDw\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\proyectoDw\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\proyectoDw\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\proyectoDw\system\BaseModel.php(839): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\proyectoDw\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\proyectoDw\app\Controllers\Registro.php(36): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\proyectoDw\system\CodeIgniter.php(933): App\Controllers\Registro->crear()
#9 C:\xampp\htdocs\proyectoDw\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registro))
#10 C:\xampp\htdocs\proyectoDw\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\proyectoDw\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\proyectoDw\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\proyectoDw\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-08-02 01:53:17 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry 'datobar@pucesi.edu.ec' for key 'correo'
[Method: POST, Route: registro/crear]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`nombre`, `correo`, `password`, `cedula`, `estado`) VALUES (:nombre:, :correo:, :password:, :cedula:, :estado:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Registro.php(36): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Registro->crear()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registro))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-08-02 01:53:17 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry 'datobar@pucesi.edu.ec' for key 'correo'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuarios` (`nombre`, `correo`, `password`, `cedula`, `estado`) VALUES (\'david\', \'datobar@pucesi.edu.ec\', \'$2y$10$wgkpweaGQ/1I9rrtdZA5J.iY3WX50SlMx4lJPMV8gETY5hqsExUJ.\', \'1234567891\', \'1\')')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuarios` (`nombre`, `correo`, `password`, `cedula`, `estado`) VALUES (\'david\', \'datobar@pucesi.edu.ec\', \'$2y$10$wgkpweaGQ/1I9rrtdZA5J.iY3WX50SlMx4lJPMV8gETY5hqsExUJ.\', \'1234567891\', \'1\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`nombre`, `correo`, `password`, `cedula`, `estado`) VALUES (:nombre:, :correo:, :password:, :cedula:, :estado:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Registro.php(36): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Registro->crear()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registro))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-08-02 01:53:17 --> [Caused by] mysqli_sql_exception: Duplicate entry 'datobar@pucesi.edu.ec' for key 'correo'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `usuarios` (`nombre`, `correo`, `password`, `cedula`, `estado`) VALUES (\'david\', \'datobar@pucesi.edu.ec\', \'$2y$10$wgkpweaGQ/1I9rrtdZA5J.iY3WX50SlMx4lJPMV8gETY5hqsExUJ.\', \'1234567891\', \'1\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuarios` (`nombre`, `correo`, `password`, `cedula`, `estado`) VALUES (\'david\', \'datobar@pucesi.edu.ec\', \'$2y$10$wgkpweaGQ/1I9rrtdZA5J.iY3WX50SlMx4lJPMV8gETY5hqsExUJ.\', \'1234567891\', \'1\')')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuarios` (`nombre`, `correo`, `password`, `cedula`, `estado`) VALUES (\'david\', \'datobar@pucesi.edu.ec\', \'$2y$10$wgkpweaGQ/1I9rrtdZA5J.iY3WX50SlMx4lJPMV8gETY5hqsExUJ.\', \'1234567891\', \'1\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`nombre`, `correo`, `password`, `cedula`, `estado`) VALUES (:nombre:, :correo:, :password:, :cedula:, :estado:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Registro.php(36): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Registro->crear()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registro))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-08-02 01:54:18 --> mysqli_sql_exception: Duplicate entry 'datobar@pucesi.edu.ec' for key 'correo' in C:\xampp\htdocs\proyectoDw\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\proyectoDw\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\proyectoDw\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\proyectoDw\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\proyectoDw\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\proyectoDw\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\proyectoDw\system\BaseModel.php(839): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\proyectoDw\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\proyectoDw\app\Controllers\Registro.php(36): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\proyectoDw\system\CodeIgniter.php(933): App\Controllers\Registro->crear()
#9 C:\xampp\htdocs\proyectoDw\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registro))
#10 C:\xampp\htdocs\proyectoDw\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\proyectoDw\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\proyectoDw\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\proyectoDw\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-08-02 01:54:18 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry 'datobar@pucesi.edu.ec' for key 'correo'
[Method: POST, Route: registro/crear]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`nombre`, `correo`, `password`, `cedula`, `estado`) VALUES (:nombre:, :correo:, :password:, :cedula:, :estado:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Registro.php(36): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Registro->crear()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registro))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-08-02 01:54:18 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry 'datobar@pucesi.edu.ec' for key 'correo'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuarios` (`nombre`, `correo`, `password`, `cedula`, `estado`) VALUES (\'david\', \'datobar@pucesi.edu.ec\', \'$2y$10$mlpcrVJkKbFyfUHVGZDp/etg62m.vev3ZxSsR0Ih0Yz6C.64O.LWW\', \'1234567891\', \'1\')')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuarios` (`nombre`, `correo`, `password`, `cedula`, `estado`) VALUES (\'david\', \'datobar@pucesi.edu.ec\', \'$2y$10$mlpcrVJkKbFyfUHVGZDp/etg62m.vev3ZxSsR0Ih0Yz6C.64O.LWW\', \'1234567891\', \'1\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`nombre`, `correo`, `password`, `cedula`, `estado`) VALUES (:nombre:, :correo:, :password:, :cedula:, :estado:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Registro.php(36): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Registro->crear()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registro))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-08-02 01:54:18 --> [Caused by] mysqli_sql_exception: Duplicate entry 'datobar@pucesi.edu.ec' for key 'correo'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `usuarios` (`nombre`, `correo`, `password`, `cedula`, `estado`) VALUES (\'david\', \'datobar@pucesi.edu.ec\', \'$2y$10$mlpcrVJkKbFyfUHVGZDp/etg62m.vev3ZxSsR0Ih0Yz6C.64O.LWW\', \'1234567891\', \'1\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuarios` (`nombre`, `correo`, `password`, `cedula`, `estado`) VALUES (\'david\', \'datobar@pucesi.edu.ec\', \'$2y$10$mlpcrVJkKbFyfUHVGZDp/etg62m.vev3ZxSsR0Ih0Yz6C.64O.LWW\', \'1234567891\', \'1\')')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuarios` (`nombre`, `correo`, `password`, `cedula`, `estado`) VALUES (\'david\', \'datobar@pucesi.edu.ec\', \'$2y$10$mlpcrVJkKbFyfUHVGZDp/etg62m.vev3ZxSsR0Ih0Yz6C.64O.LWW\', \'1234567891\', \'1\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`nombre`, `correo`, `password`, `cedula`, `estado`) VALUES (:nombre:, :correo:, :password:, :cedula:, :estado:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Registro.php(36): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Registro->crear()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registro))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-08-02 01:54:33 --> mysqli_sql_exception: Duplicate entry 'datobar@pucesi.edu.ec' for key 'correo' in C:\xampp\htdocs\proyectoDw\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\proyectoDw\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\proyectoDw\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\proyectoDw\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\proyectoDw\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\proyectoDw\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\proyectoDw\system\BaseModel.php(839): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\proyectoDw\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\proyectoDw\app\Controllers\Registro.php(36): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\proyectoDw\system\CodeIgniter.php(933): App\Controllers\Registro->crear()
#9 C:\xampp\htdocs\proyectoDw\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registro))
#10 C:\xampp\htdocs\proyectoDw\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\proyectoDw\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\proyectoDw\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\proyectoDw\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-08-02 01:54:33 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry 'datobar@pucesi.edu.ec' for key 'correo'
[Method: POST, Route: registro/crear]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`nombre`, `correo`, `password`, `cedula`, `estado`) VALUES (:nombre:, :correo:, :password:, :cedula:, :estado:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Registro.php(36): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Registro->crear()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registro))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-08-02 01:54:33 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry 'datobar@pucesi.edu.ec' for key 'correo'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuarios` (`nombre`, `correo`, `password`, `cedula`, `estado`) VALUES (\'david\', \'datobar@pucesi.edu.ec\', \'$2y$10$qvd8.DnmIXzXiu5pJICiy.Ja2y.at2xfCmSrqHM5ZeX7/69.vyzfy\', \'1234567891\', \'1\')')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuarios` (`nombre`, `correo`, `password`, `cedula`, `estado`) VALUES (\'david\', \'datobar@pucesi.edu.ec\', \'$2y$10$qvd8.DnmIXzXiu5pJICiy.Ja2y.at2xfCmSrqHM5ZeX7/69.vyzfy\', \'1234567891\', \'1\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`nombre`, `correo`, `password`, `cedula`, `estado`) VALUES (:nombre:, :correo:, :password:, :cedula:, :estado:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Registro.php(36): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Registro->crear()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registro))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-08-02 01:54:33 --> [Caused by] mysqli_sql_exception: Duplicate entry 'datobar@pucesi.edu.ec' for key 'correo'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `usuarios` (`nombre`, `correo`, `password`, `cedula`, `estado`) VALUES (\'david\', \'datobar@pucesi.edu.ec\', \'$2y$10$qvd8.DnmIXzXiu5pJICiy.Ja2y.at2xfCmSrqHM5ZeX7/69.vyzfy\', \'1234567891\', \'1\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuarios` (`nombre`, `correo`, `password`, `cedula`, `estado`) VALUES (\'david\', \'datobar@pucesi.edu.ec\', \'$2y$10$qvd8.DnmIXzXiu5pJICiy.Ja2y.at2xfCmSrqHM5ZeX7/69.vyzfy\', \'1234567891\', \'1\')')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuarios` (`nombre`, `correo`, `password`, `cedula`, `estado`) VALUES (\'david\', \'datobar@pucesi.edu.ec\', \'$2y$10$qvd8.DnmIXzXiu5pJICiy.Ja2y.at2xfCmSrqHM5ZeX7/69.vyzfy\', \'1234567891\', \'1\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`nombre`, `correo`, `password`, `cedula`, `estado`) VALUES (:nombre:, :correo:, :password:, :cedula:, :estado:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Registro.php(36): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Registro->crear()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registro))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-08-02 01:54:45 --> mysqli_sql_exception: Duplicate entry 'datobar@pucesi.edu.ec' for key 'correo' in C:\xampp\htdocs\proyectoDw\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\proyectoDw\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\proyectoDw\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\proyectoDw\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\proyectoDw\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\proyectoDw\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\proyectoDw\system\BaseModel.php(839): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\proyectoDw\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\proyectoDw\app\Controllers\Registro.php(36): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\proyectoDw\system\CodeIgniter.php(933): App\Controllers\Registro->crear()
#9 C:\xampp\htdocs\proyectoDw\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registro))
#10 C:\xampp\htdocs\proyectoDw\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\proyectoDw\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\proyectoDw\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\proyectoDw\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-08-02 01:54:45 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry 'datobar@pucesi.edu.ec' for key 'correo'
[Method: POST, Route: registro/crear]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`nombre`, `correo`, `password`, `cedula`, `estado`) VALUES (:nombre:, :correo:, :password:, :cedula:, :estado:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Registro.php(36): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Registro->crear()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registro))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-08-02 01:54:45 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry 'datobar@pucesi.edu.ec' for key 'correo'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuarios` (`nombre`, `correo`, `password`, `cedula`, `estado`) VALUES (\'david012\', \'datobar@pucesi.edu.ec\', \'$2y$10$Tc8SlcLN6vQpcib77Hi2e.iESmWrQm9drsKjl7dWmZHvc/jurBlmi\', \'1234567891\', \'1\')')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuarios` (`nombre`, `correo`, `password`, `cedula`, `estado`) VALUES (\'david012\', \'datobar@pucesi.edu.ec\', \'$2y$10$Tc8SlcLN6vQpcib77Hi2e.iESmWrQm9drsKjl7dWmZHvc/jurBlmi\', \'1234567891\', \'1\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`nombre`, `correo`, `password`, `cedula`, `estado`) VALUES (:nombre:, :correo:, :password:, :cedula:, :estado:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Registro.php(36): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Registro->crear()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registro))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-08-02 01:54:45 --> [Caused by] mysqli_sql_exception: Duplicate entry 'datobar@pucesi.edu.ec' for key 'correo'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `usuarios` (`nombre`, `correo`, `password`, `cedula`, `estado`) VALUES (\'david012\', \'datobar@pucesi.edu.ec\', \'$2y$10$Tc8SlcLN6vQpcib77Hi2e.iESmWrQm9drsKjl7dWmZHvc/jurBlmi\', \'1234567891\', \'1\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuarios` (`nombre`, `correo`, `password`, `cedula`, `estado`) VALUES (\'david012\', \'datobar@pucesi.edu.ec\', \'$2y$10$Tc8SlcLN6vQpcib77Hi2e.iESmWrQm9drsKjl7dWmZHvc/jurBlmi\', \'1234567891\', \'1\')')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuarios` (`nombre`, `correo`, `password`, `cedula`, `estado`) VALUES (\'david012\', \'datobar@pucesi.edu.ec\', \'$2y$10$Tc8SlcLN6vQpcib77Hi2e.iESmWrQm9drsKjl7dWmZHvc/jurBlmi\', \'1234567891\', \'1\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`nombre`, `correo`, `password`, `cedula`, `estado`) VALUES (:nombre:, :correo:, :password:, :cedula:, :estado:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Registro.php(36): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Registro->crear()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registro))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-08-02 02:21:54 --> Error connecting to the database: mysqli_sql_exception: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\nuevo-proyectoDW\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\nuevo-proyectoDW\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'proyectodw', 3307, '', 0)
#1 C:\xampp\htdocs\nuevo-proyectoDW\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\nuevo-proyectoDW\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\nuevo-proyectoDW\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\nuevo-proyectoDW\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\nuevo-proyectoDW\system\BaseModel.php(839): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\nuevo-proyectoDW\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\nuevo-proyectoDW\app\Controllers\Registro.php(36): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\nuevo-proyectoDW\system\CodeIgniter.php(933): App\Controllers\Registro->crear()
#9 C:\xampp\htdocs\nuevo-proyectoDW\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registro))
#10 C:\xampp\htdocs\nuevo-proyectoDW\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\nuevo-proyectoDW\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\nuevo-proyectoDW\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\nuevo-proyectoDW\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\nuevo-proyectoDW\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\nuevo-proyectoDW\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\nuevo-proyectoDW\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\nuevo-proyectoDW\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#3 C:\xampp\htdocs\nuevo-proyectoDW\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\nuevo-proyectoDW\system\BaseModel.php(839): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\nuevo-proyectoDW\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#6 C:\xampp\htdocs\nuevo-proyectoDW\app\Controllers\Registro.php(36): CodeIgniter\Model->insert(Array)
#7 C:\xampp\htdocs\nuevo-proyectoDW\system\CodeIgniter.php(933): App\Controllers\Registro->crear()
#8 C:\xampp\htdocs\nuevo-proyectoDW\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registro))
#9 C:\xampp\htdocs\nuevo-proyectoDW\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\nuevo-proyectoDW\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\nuevo-proyectoDW\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\nuevo-proyectoDW\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-08-02 02:21:54 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión
[Method: POST, Route: registro/crear]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`nombre`, `correo`, `password`, `cedula`, `estado`) VALUES (:nombre:, :correo:, :password:, :cedula:, :estado:)', [...], false)
 3 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 6 APPPATH\Controllers\Registro.php(36): CodeIgniter\Model->insert([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Registro->crear()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registro))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-08-02 02:22:34 --> Error connecting to the database: mysqli_sql_exception: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\nuevo-proyectoDW\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\nuevo-proyectoDW\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'proyectodw', 3307, '', 0)
#1 C:\xampp\htdocs\nuevo-proyectoDW\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\nuevo-proyectoDW\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\nuevo-proyectoDW\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\nuevo-proyectoDW\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\nuevo-proyectoDW\system\BaseModel.php(839): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\nuevo-proyectoDW\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\nuevo-proyectoDW\app\Controllers\Registro.php(36): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\nuevo-proyectoDW\system\CodeIgniter.php(933): App\Controllers\Registro->crear()
#9 C:\xampp\htdocs\nuevo-proyectoDW\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registro))
#10 C:\xampp\htdocs\nuevo-proyectoDW\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\nuevo-proyectoDW\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\nuevo-proyectoDW\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\nuevo-proyectoDW\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\nuevo-proyectoDW\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\nuevo-proyectoDW\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\nuevo-proyectoDW\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\nuevo-proyectoDW\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#3 C:\xampp\htdocs\nuevo-proyectoDW\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\nuevo-proyectoDW\system\BaseModel.php(839): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\nuevo-proyectoDW\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#6 C:\xampp\htdocs\nuevo-proyectoDW\app\Controllers\Registro.php(36): CodeIgniter\Model->insert(Array)
#7 C:\xampp\htdocs\nuevo-proyectoDW\system\CodeIgniter.php(933): App\Controllers\Registro->crear()
#8 C:\xampp\htdocs\nuevo-proyectoDW\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registro))
#9 C:\xampp\htdocs\nuevo-proyectoDW\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\nuevo-proyectoDW\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\nuevo-proyectoDW\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\nuevo-proyectoDW\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-08-02 02:22:34 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión
[Method: POST, Route: registro/crear]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`nombre`, `correo`, `password`, `cedula`, `estado`) VALUES (:nombre:, :correo:, :password:, :cedula:, :estado:)', [...], false)
 3 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 6 APPPATH\Controllers\Registro.php(36): CodeIgniter\Model->insert([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Registro->crear()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registro))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-08-02 02:23:37 --> Error connecting to the database: mysqli_sql_exception: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\nuevo-proyectoDW\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\nuevo-proyectoDW\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'proyectodw', 3307, '', 0)
#1 C:\xampp\htdocs\nuevo-proyectoDW\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\nuevo-proyectoDW\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\nuevo-proyectoDW\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\nuevo-proyectoDW\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\nuevo-proyectoDW\system\BaseModel.php(839): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\nuevo-proyectoDW\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\nuevo-proyectoDW\app\Controllers\Registro.php(36): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\nuevo-proyectoDW\system\CodeIgniter.php(933): App\Controllers\Registro->crear()
#9 C:\xampp\htdocs\nuevo-proyectoDW\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registro))
#10 C:\xampp\htdocs\nuevo-proyectoDW\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\nuevo-proyectoDW\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\nuevo-proyectoDW\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\nuevo-proyectoDW\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\nuevo-proyectoDW\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\nuevo-proyectoDW\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\nuevo-proyectoDW\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\nuevo-proyectoDW\system\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#3 C:\xampp\htdocs\nuevo-proyectoDW\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\nuevo-proyectoDW\system\BaseModel.php(839): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\nuevo-proyectoDW\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#6 C:\xampp\htdocs\nuevo-proyectoDW\app\Controllers\Registro.php(36): CodeIgniter\Model->insert(Array)
#7 C:\xampp\htdocs\nuevo-proyectoDW\system\CodeIgniter.php(933): App\Controllers\Registro->crear()
#8 C:\xampp\htdocs\nuevo-proyectoDW\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registro))
#9 C:\xampp\htdocs\nuevo-proyectoDW\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\nuevo-proyectoDW\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\nuevo-proyectoDW\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\nuevo-proyectoDW\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-08-02 02:23:37 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión
[Method: POST, Route: registro/crear]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2340): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`nombre`, `correo`, `password`, `cedula`, `estado`) VALUES (:nombre:, :correo:, :password:, :cedula:, :estado:)', [...], false)
 3 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 6 APPPATH\Controllers\Registro.php(36): CodeIgniter\Model->insert([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Registro->crear()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Registro))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
